lib/project_types/theme/commands/package.rb

```diff
@@ -18,11 +18,15 @@ class Package < ShopifyCLI::Command::SubCommand
         release-notes.md
       ]
 
+      ZIP = "zip"
+      SEVEN_ZIP = "7z"
```

Comment:
how do we feel about using `SEVEN` cause a number can't be the first letter of the variable name?

Reply:
+1 from my side :)